### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by dvdplm** [arkworks-rs/algebra#813]
First of all, thank you for your hard work. ❤️ 

I am trying to optimise build times and the ark-* crates at v0.4.2 are causing both `syn` v1 and v2 to get pulled in (by way of `zeroize`). This is fixed on `main` so I came here to ask when we can look forward to a new release. Do holler if I can help out in any way.

**Comment by rillian** [comment]
`cargo audit` also warns about the criterion 0.4 specification in the arkworks 0.4.2 crates; making a new release would address that as well (#653).

**Comment by w-flo** [comment]
> [..] are causing both `syn` v1 and v2 to get pulled in (by way of `zeroize`). This is fixed on `main` [..]

I believe they still depend on `derivative` on `main`, right? `derivative` seems to be unmaintained currently, and it still depends on `syn` v1 as well.

**Comment by hrxi** [comment]
A new release would also be good for using this crate on WASM, since compilation was broken there, but fixed with #794.

**Comment by xlc** [comment]
Any updates about new release?

**Comment by TalDerei** [comment]
update on new release that integrates https://github.com/arkworks-rs/algebra/issues/578?

**Comment by mmagician** [comment]
0.5.0 includes the changes in https://github.com/arkworks-rs/algebra/pull/828

**Comment by cre-mer** [comment]
@w-flo Is there any update on the `derivative` dependency? Will it be replaced at some point?

**Comment by hrxi** [comment]
> @w-flo Is there any update on the `derivative` dependency? Will it be replaced at some point?

Is it about something else than the "unmaintained" status? If not, why would you change from a crate that has no problems? Being unmaintained is not a problem if it works.

**Comment by cre-mer** [comment]
@hrxi fair point. Issues can be found and raised to unmaintained crates, but they will remain unfixed. Maintained crates are not a security guarantee either. There's just a higher chance that bugs get fixed when they're found and raised.
